src/main/webapp/i18n/de/student-dashboard.json

```
@@ -169,6 +169,7 @@
                 "quizNotStartedShort": "Nicht begonnen",
                 "userNotParticipatedShort": "Nicht teilgenommen",
                 "userParticipatingShort": "Aktuelle Teilnahme",
+                "userWaitingForDueDateShort": "Teilgenommen, warte auf Frist",
```

```suggestion
                "userWaitingForDueDateShort": "Abgegeben, warte auf Frist",
```
I'd recommend this version since it indicates that the exercise has been submitted. "Teilgenommen" would only indicate that you've participated, but not necessarily submitted.